EC-84: Revise Configuration, Customers, eContent, Login and Logging (#5)
ezsystems/commerce-documentation: docs/guide/logging/tokencontroller_logging.md

# TokenController logging

> If a token is processed using the [TokenController](../user_management/token/tokencontroller/tokencontroller.md),

Reviewer: ```suggestion
If a token is processed with the [TokenController](../user_management/token/tokencontroller/tokencontroller.md),
```

> it is logged in a file or database. By default the configuration uses database logging:

Reviewer: ```suggestion
it is logged in a file or database. By default, the configuration uses database logging:
```

## Configuration

> File and database logging is activated by adding the `pushHandler()` method call with the `gdpr_log_handler.doctrine` and/or `gdpr_log_handler.file` service as an argument:

Reviewer: ```suggestion
You activate the file and database logging by adding the `pushHandler()` method call with the `gdpr_log_handler.doctrine` and/or `gdpr_log_handler.file` service as an argument:
```
